src/Esfa.Vacancy.Register.Api/Web.config

```diff
@@ -15,6 +15,7 @@
     <add key="UnobtrusiveJavaScriptEnabled" value="true" />
     <add key="ApprenticeshipProviderElements" value="20" />
     <add key="LoggingRedisKey" value="logstash" />
+    <add key="DAS_AVMSPLUSCONNECTIONSTRING" value="" />
```

Comment:
Yes, I missed changing that in the web.config. 
I wanted to have a think about how these settings are named as well. All caps seems bit loud and difficult to read. I would like to see grouping like it is done in webpages:version. So for this one, I suggest we could do something like "_connectionstring:avmcplusdb_" and we could add more to it in future for ex: "_connectionstring:azurestorage_" or "_uri:internal-api-url_" etc. 
Thoughts?